src/index.tsx

```diff
@@ -12,9 +12,15 @@ const LazyConfig = React.lazy(async () => import('./pages/config.jsx'))
 const LazyHelperUi = React.lazy(async () => import('./pages/helper-ui.jsx'))
 const LazyRedirectPage = React.lazy(async () => import('./pages/redirect-page.jsx'))
 const LazyInterstitial = React.lazy(async () => import('./pages/redirects-interstitial.jsx'))
+const LazyServiceWorkerErrorPage = React.lazy(async () => import('./pages/errors/no-service-worker.jsx'))
+
+let ErrorPage: null | React.LazyExoticComponent<() => React.JSX.Element> = null
+if (navigator.serviceWorker == null) {
+  ErrorPage = LazyServiceWorkerErrorPage
+}
 
 const routes: Route[] = [
-  { default: true, component: LazyHelperUi },
+  { default: true, component: ErrorPage ?? LazyHelperUi },
```

Comment:
gives us the ability to easily set different error pages for other conditions, without fronting the loading of all the errors for all users.